src/components/content/workflows/Workflows.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Row, Space, Table, Tag } from 'antd';
+import { ColumnsType } from 'antd/es/table';
+import { ApiError, Response, WorkflowService, WorkFlowTask } from '../../../xpanse-api/generated';
+import {
+    CheckCircleOutlined,
+    CloseCircleOutlined,
+    FullscreenOutlined,
+    InfoCircleOutlined,
+    SyncOutlined,
+} from '@ant-design/icons';
+import { WorkflowsTip } from './WorkflowsTip';
+import useAllTasksQuery from './query/useAllTasksQuery';
+import { useMutation } from '@tanstack/react-query';
+import { formatDate } from '../../utils/DateUtils';
+
+function Workflows(): React.JSX.Element {
+    const [tipMessage, setTipMessage] = useState<string>('');
+    const [tipType, setTipType] = useState<'error' | 'success' | undefined>(undefined);
+    const [isRefresh, setIsRefresh] = useState(false);
+    const [todoTasks, setTodoTasks] = useState<WorkFlowTask[]>([]);
+
+    const tasksQuery = useAllTasksQuery(undefined);
+
+    useEffect(() => {
+        const workflowTasks: WorkFlowTask[] | undefined = tasksQuery.data;
+        if (workflowTasks !== undefined && workflowTasks.length > 0) {
+            setTodoTasks(workflowTasks);
+        } else {
+            setTodoTasks([]);
+        }
+    }, [tasksQuery.data, tasksQuery.isSuccess]);
+
+    useEffect(() => {
+        if (tasksQuery.error instanceof ApiError && tasksQuery.error.body && 'details' in tasksQuery.error.body) {
+            const response: Response = tasksQuery.error.body as Response;
+            getTipInfo('error', response.details.join());
+        } else if (tasksQuery.error instanceof Error) {
+            getTipInfo('error', tasksQuery.error.message);
+        }
+        setIsRefresh(false);
+    }, [tasksQuery.error]);
+
+    const completeFailedTasksQuery = useMutation({
+        mutationFn: (taskId: string) => WorkflowService.manageFailedOrder(taskId, true),
+        onSuccess: () => {
+            getTipInfo('success', 'handler failed task complete.');
+            setIsRefresh(false);
+            void tasksQuery.refetch();
+        },
+        onError: (error: Error) => {
+            setIsRefresh(false);
+            if (error instanceof ApiError && error.body && 'details' in error.body) {
+                const response: Response = error.body as Response;
+                getTipInfo('error', response.details.join());
+            } else {
+                getTipInfo('error', error.message);
+            }
+        },
+    });
+
+    const closeFailedTasksQuery = useMutation({
+        mutationFn: (taskId: string) => WorkflowService.manageFailedOrder(taskId, false),
+        onSuccess: () => {
+            getTipInfo('success', 'handler failed task complete.');
```

Comment:
Failed workflow is closed successfully. Please create a new request if necessary.